README.md

```diff
@@ -8,18 +8,41 @@ Create a coc extension quickly 🚀
 
 ```sh
 npx create-coc-extension [extension-name]
+# or
+npm exec create-coc-extension [extension-name]
+cd [extension-name]
+npm install
+npm run build
 ```
 
 ### npm
 
 ```sh
+npm create coc-extension [extension-name]
+# or
 npm init coc-extension [extension-name]
+cd [extension-name]
+npm install
+npm run build
 ```
 
-then cd to your extension root:
+### yarn
 
```

Comment:
No more yarn please, I've removed yarn in https://github.com/fannheyward/create-coc-extension/commit/759d4172f9df96cb956fb15d08efa8df356c961f, because yarn 1 is dead and yarn 3 is always used by-project level.